blocks/image-collage/image-collage.css

```diff
@@ -31,7 +52,56 @@ main .image-collage[class*='boxy-col-'].image-with-title > div {
   flex-basis: calc(50% - 16px);
 }
 
-main .image-collage[class*='boxy-col-']>div>div:first-of-type {
+main .image-collage.icon-caption > div > div:first-of-type,
+main .image-collage.icon-title-list > div > div:first-of-type {
+  padding-top: unset;
+  padding-bottom: unset;
+  padding-right: 1rem;
+}
+
+main .image-collage.icon-title-text > div > div:first-of-type {
+  padding-top: unset;
+  padding-bottom: unset;
+  display: flex;
+  margin-bottom: 0.5rem;
+}
+
+main .image-collage.icon-title-text > div > div:first-of-type h4{
+  padding-left: 1rem;
+  line-height: 1.875rem;
+  font-size: var(--heading-font-size-s);
+  margin: unset;
+}
+
+main .image-collage.icon-title-list > div > div p {
+  font-size: var(--body-font-size-m);
+  font-weight: var(--font-weight-bold);
+  line-height: 1.5rem;
+}
+
+main .image-collage.icon-title-list > div > div ul {
+  list-style-type: none;
+  padding: unset;
+  margin: unset;
+}
+
+main .image-collage.icon-title-list > div > div li {
+  margin-bottom: 9px;
+  text-indent: -9px;
+  margin-left: 1rem;
+  line-height: 1.1rem;
+}
+
+main .image-collage.icon-title-list > div > div li::before {
+  content: "\2022";
```

Comment:
Can't we copy-paste the actual character?

Reply:
Done, you can check. 
(Personally, I prefer the Unicode to the bullet mark ^ ^)

Reply:
My thinking was that I had to google what the unicode meant, so why not just have it there anyway, but thanks for changing it 👍